src/todos/todos.service.ts

```diff
@@ -22,4 +22,9 @@ export class TodosService {
     }
     return foundTodo;
   }
+
+  async deleteTodoById(id: string): Promise<void> {
```

Comment:
```suggestion
  async deleteOne(id: string): Promise<void> {
```

No need to specify `todo` as this is in todosService